Sources/Turf/GeoJSON.swift

```diff
@@ -98,11 +98,26 @@ public protocol ForeignMemberContainer {
     var foreignMembers: JSONObject { get set }
 }
 
+/**
+ Key to pass to populate a `userInfo` dictionary, which is passed to the `JSONDecoder` or `JSONEncoder` to enable processing foreign members.
+*/
+public extension CodingUserInfoKey {
+    /**
+     Indicates if coding of foreign members is enabled.
+     
+     Boolean flag to enable coding. Default (or missing) value is to ignore foreign members.
```

Comment:
Especially because of this default, we should make the key as discoverable as possible. Let’s mention it in the documentation comments for ForeignMemberContainer and the types that are most likely to contain foreign members: Feature, FeatureCollection, and GeoJSONObject.

Reply:
Added notes for `GeoJSONObject` and `ForeignMemberContainer.foreignMembers`. `Feature` and `FeatureCollection` have doc automatically imported from `ForeignMemberContainer`.